packages/dds/tree/src/feature-libraries/flex-tree/lazyField.ts

```diff
@@ -331,15 +331,6 @@ export class LazySequence<TTypes extends FlexAllowedTypes>
 		fieldEditor.remove(index, 1);
 	}
```

Comment:
Could you add a quick TODO in here (or maybe in the interface declaration, or wherever it makes sense) saying something like "TODO: Remove remaining list mutation methods and have callers access `sequenceEditor` directly instead." Given the new state after this PR, a casual observer might be confused as to why we've implemented an analog to every method in the simple-tree API _except_ for `removeRange`.

Reply:
done (on interface)